README.md

```diff
@@ -8,83 +8,44 @@ iOS Playbook 📚
+|-------------------------------|------------------------------- |
+| **Booking** <br> Face to face appointment booking. | Witold Skibniewski • [GitHub](https://github.com/mr-v)|
+| **Consultation** <br> Flows for all the consultation like prescriptions and video consultation. | Adrian Śliwa • [Twitter](https://twitter.com/adiki91) • [GitHub](https://github.com/adiki) <br> Chitra Kotwani • [Twitter](https://twitter.com/chitrakotwani) • [GitHub](https://github.com/chitrakotwani)|
+|**Core Experience** <br> General user experience in the app. | Emese Toth • [GitHub](https://github.com/emeseuk) <br> Giorgos Tsiapaliokas • [GitHub](https://github.com/gtsiap) <br> Sergey Shulga • [Twitter](https://twitter.com/SergDort) • [GitHub](https://github.com/sergdort) <br> Yuri Karabatov • [Twitter](https://twitter.com/karabatov) • [GitHub](https://github.com/karabatov)|
+|**Native Apps Platform** <br> Engineering work like tooling, CI and development processes. | David Rodrigues • [Twitter](https://twitter.com/dmcrodrigues) • [GitHub](https://github.com/dmcrodrigues) <br> Ilya Puchka • [Twitter](https://twitter.com/ilyapuchka) • [GitHub](https://github.com/ilyapuchka) <br> Yasuhiro Inami • [Twitter](https://twitter.com/inamiy) • [GitHub](https://github.com/inamiy) <br> Martin Nygren • [GitHub](https://github.com/zzcgumn) <br>  Olivier Halligon • [Twitter](https://twitter.com/aligatr) • [GitHub](https://github.com/AliSoftware)|
+|**GP @ Hand** <br> End to end journey for NHS registration. | James Birtwell • [GitHub](https://github.com/jimmybee) |
+|**Healthcheck** <br> Overview of your health using a 3D body model (avatar). | Ben Henshall • [Twitter](https://twitter.com/ben_henshall?lang=en) • [GitHub](https://github.com/Ben-Henshall) <br> Julien Ducret • [Twitter](https://github.com/brocoo) • [GitHub](https://github.com/brocoo) |
+| **Monitor** <br> Monitoring health metrics like activity, blood, urine, ... | Anders Ha • [Twitter](https://twitter.com/_andersha) • [GitHub](https://github.com/andersio) <br> Daniel Haight • [GitHub](https://github.com/Daniel1of1) <br> Daniel Spindelbauer • [Twitter](https://twitter.com/sdaniel55) • [GitHub](https://github.com/sdaniel55) <br> Diego  Petrucci • [Twitter](https://twitter.com/diegopetrucci) • [GitHub](https://github.com/diegopetrucci) <br> Joshua Simmons • [GitHub](https://github.com/j531) |
+| **Prescriptions** <br> Precriptions functionality. | Adam Borek • [Twitter](https://twitter.com/TheAdamBorek) • [GitHbu](https://github.com/TheAdamBorek) <br> Konrad Muchowicz • [GitHub](https://github.com/konrad-em)
+| **Professional Services** | Simon Cass • [Twitter](https://twitter.com/codercass) • [GitHub](https://github.com/scass91)|
+| **SDK** <br> Develop and mantain the SDK frameworks. | Viorel Mihalache • [Twitter](https://twitter.com/viorelMO) • [GitHub](https://github.com/viorel15) |
+| **Tenancy & Features** | Anil Puttabuddhi • [GitHub](https://github.com/anilputtabuddhi) |
```

Comment:
No squad description like for the others?

Reply:
I still have to know what this is about 😅